tests/rkt_run_pod_manifest_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
 )
 
-const baseAppName = "coreos.com/rkt-inspect"
+const baseAppName = "rkt-inspect"
```

Comment:
@jonboulle Nope, it's the app name, should be ACName type, cannot contains `/` `.` :)